homework_10,11.py

```diff
@@ -0,0 +1,68 @@
+# 1. Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+from datetime import datetime
+
+now = datetime.now()
+current_time = now.strftime("%H:%M:%S")
+
+def my_decorator(func):
+    def deco_func(*args, **kwargs):
+        now = datetime.now()
+        current_time = now.strftime("%H:%M:%S")
+        print(f"Function name: {func}")
+        print(f"Function call time: {current_time}")
+        result = func(*args, **kwargs)
+        return round(result, 0)
```

Comment:
This code will raise an exception if you decorate a function that returns a string, for example.
Leave it just as `return result`